### PR TITLE
Stop submitting logs if rate limit is reached

### PR DESCRIPTION
**Pull request by ShivamPokhriyal**
This PR will break out of log submission loop as soon as we encounter a rate limit error.

### CODE REVIEW

File: app/assets/locales/android_translatable_strings.txt [hunk 1]
```diff
@@ -650,6 +650,9 @@ notification.logger.serialized.detail=CommCare had a problem communicating devic
 notification.logger.error.title=Error submitting logs!
 notification.logger.error.detail=CommCare ran into an error trying to submit your device logs to the server! Please contact CommCare Support if this persists.
 
+notification.logger.rate.limited.title=Log Submission rate limit reached.
```
Comment (shubham1g5) [inline app/assets/locales/android_translatable_strings.txt:653]: nit: rename `logger` - `logs` in key

**Review by shubham1g5 — APPROVED** [review]

File: app/assets/locales/android_translatable_strings.txt (hunk 1 above)
Comment (ShivamPokhriyal) [reply app/assets/locales/android_translatable_strings.txt:653]: Do you mean `notification.logger.rate.limited.title` to `notification.logs.rate.limited.title`? 
Additionally, I tried to be consistent with other keys in enum(all of them have this `notification.logger` prefix).

Comment (shubham1g5) [reply app/assets/locales/android_translatable_strings.txt:653]: got it, no need to change in that case.

**Comment by dimagimon** [comment]
[32% (0.0%) vs master 32%](https://jenkins.dimagi.com/job/commcare-android-pr-job-builder/462115/)

**Pull request merged**